.github/renovate.json

```diff
@@ -32,5 +32,6 @@
       "allowedVersions": "<5.0.0"
     }
   ],
-  "ignorePaths": ["**/dist-dynamic/**"]
+  "ignorePaths": ["**/dist-dynamic/**"],
+  "gitIgnoredAuthors": ["backstage-goalie"]
```

Comment:
This has to be the email address of the bot, as git uses these to compare commits.

See: 
- https://docs.renovatebot.com/configuration-options/#gitignoredauthors
- https://stackoverflow.com/questions/66908568/how-to-correctly-set-a-github-app-committer-to-show-the-app-created-the-commit

Reply:
Ah, alright, not sure how to figure that out but will dig into it. Thanks for taking the time to review this 👍

Reply:
@benjdlambert @freben could either of you help me with this? Having this in place would help with our Renovate process in this repo.

Reply:
@Rugvip do you know what the email of the `backstage-goalie` bot is/

Reply:
Hey @awanlin, does this help?

```
community-plugins $ git shortlog --summary --numbered --email | grep goalie
    32  backstage-goalie[bot] <97962292+backstage-goalie[bot]@users.noreply.github.com>
```